components/AlbumDetails.vue

```diff
@@ -41,34 +51,26 @@ const toggleExpandedAlbum = (albumReference: string) => {
               params: { id: albumId },
             }"
           >
-            <ButtonStyled intent="secondary" class="h-full aspect-square">
-              <NuxtIcon name="icon.link" />
-            </ButtonStyled>
+            <ButtonStyled icon="icon.link" icon-only></ButtonStyled>
           </CopyToClipboard>
         </div>
       </div>
     </header>
-    <p v-if="album.children" class="p-2">
-      {{ t("collection.album-count", album.children.length) }}
+    <p v-if="album.children" class="py-2 text-label-3">
+      <template v-if="childAlbums.length > 0">
```

Comment:
What if it has both? What if an album contains both albums and tracks as children? We have that at least once.

Reply:
Will be fixed in an upcoming PR. There's another album, "Påskecamp 2024", where we again have an album containing tracks and albums.